cfpb/consumerfinance.gov
# Remove site history template override

Way back in #6060 in October 2020, we decided to override the site history report template to remove the Excel export option; the report was so big it caused server errors when you tried to export it as an XSLX file. Fast forward a year and a half, and the base site history report template has changed enough that our override doesn't work in Wagtail 2.15. Since literally no one has downloaded this report in 1.5 years, let's just get rid of our override and live with a buggy (but totally unused) Excel export option.

---

## Removals

- Site history report template override

## How to test this PR

1. Go to `/admin/reports/site-history/` in Wagtail 2.15 and verify that the report loads (there should be about 2900 pages)

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
